lib/v2/5eplay/index.js

```diff
@@ -0,0 +1,100 @@
+const cheerio = require('cheerio');
+const zlib = require('zlib');
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const config = require('@/config').value;
+const { getAcwScV2ByArg1 } = require('./utils');
+
+module.exports = async (ctx) => {
+    const rootUrl = 'https://csgo.5eplay.com/';
+    const apiUrl = `${rootUrl}api/article?page=1&type_id=0&time=0&order_by=0`;
+    const articleUrl = `${rootUrl}article`;
+
+    const { data: response } = await got({
+        method: 'get',
+        url: apiUrl,
+    });
+
+    // get acw_sc__v2
+    const acw_sc__v2 = await ctx.cache.tryGet(
+        articleUrl,
+        async () => {
+            // Zlib Z_BUF_ERROR: unexpected end of file, should close decompress
+            const detailResponse = await got(
+                {
+                    method: 'get',
+                    url: articleUrl,
+                },
+                {
+                    decompress: false,
+                }
+            );
+
+            const unzipData = zlib.createUnzip({
+                chunkSize: 20 * 1024,
+            });
+            unzipData.write(detailResponse.body);
+
+            let acw_sc__v2 = '';
+            for await (const data of unzipData) {
+                const strData = data.toString();
+                const matches = strData.match(/var arg1='(.*?)';/);
+                if (matches) {
+                    acw_sc__v2 = getAcwScV2ByArg1(matches[1]);
+                    break;
+                }
+            }
+            return acw_sc__v2;
+        },
+        Math.min(config.cache.routeExpire, 25 * 60),
+        false
+    );
+
+    const items = await Promise.all(
+        response.data.list.map((item) =>
+            ctx.cache.tryGet(item.jump_link, async () => {
+                if (!acw_sc__v2) {
+                    return {
+                        title: item.title,
+                        description: item.title + (item.images?.[0] ? `<img src="${item.images[0]}" referrerpolicy="no-referrer">` : ''),
```

Comment:
```suggestion
                        description: item.title + (item.images?.[0] ? `<img src="${item.images[0]}"` : ''),
```
No need to add referer policy. This is handled by RSSHub
https://github.com/DIYgod/RSSHub/blob/db56ef5fcade70863fad3062d63046b2d7b83470/lib/middleware/parameter.js#L137-L139